### PR TITLE
add controls to empty folder on server

### LINKED ISSUE #966: add a simple option to empty DeltaChat and/or INBOX on the server
the option will go to advanced/experimental and there should be a harsh warning. 

@adbenitez o/

### PR DESCRIPTION
**Pull request by r10s**
needs https://github.com/deltachat/deltachat-core/pull/691 to be merged first, closes #966

### CODE REVIEW

File: src/org/thoughtcrime/securesms/preferences/AdvancedPreferenceFragment.java [hunk 1]
```diff
@@ -270,4 +277,41 @@ private void exportKeys() {
...
+
+  /***********************************************************************************************
+   * Empty server folder
+   **********************************************************************************************/
+
+  private class EmptyServerFoldersListener implements Preference.OnPreferenceClickListener {
+    @Override
+    public boolean onPreferenceClick(Preference preference) {
+      View gl = View.inflate(getActivity(), R.layout.empty_folder_options, null);
+      CheckBox inbox_cb = gl.findViewById(R.id.empty_inbox_folder);
```
Comment (cyBerta) [inline src/org/thoughtcrime/securesms/preferences/AdvancedPreferenceFragment.java]: from a codestyle point of view I would prefer camelCase

**Comment by cyBerta** [comment]
besides the CamelCase/snake_case inconsistency the PR looks good to me.

**Comment by r10s** [comment]
this is what it looks like now:

Screen Shot 2019-06-25 at 16 13 37

**Comment by adbenitez** [comment]
once it is merged transifex updates automatically? I would love to translate this before it is released.

**Comment by cyBerta** [comment]
> once it is merged transifex updates automatically? I would love to translate this before it is released.

No, the strings are hard coded. @r10s Even though it's experimental I think it makes sense to have those strings translatable, especially if this feature targets at a spanish speaking user group (amongst others of course).

**Comment by r10s** [comment]
as the wording is already roughly discussed, i think we can add the strings to transifex. i'll do a pr.

**Pull request merged**